.github/workflows/deploy.yml

```
@@ -76,63 +76,63 @@ jobs:
         run: ${{ github.event.repository.name }}/package.sh
 
       - name: Upload binaries to release of TGZ
-        uses: svenstaro/upload-release-action@2.11.4
+        uses: svenstaro/upload-release-action@9f7ceffe1410bbe7e95b7902af89c91da4d1dc0a  # 2.11.4
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           file: ${{ runner.workspace }}/eccube-${{ github.event.release.tag_name }}.tar.gz
```

actionlint (1.7.11)

[error] property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)